akka-persistence/src/test/scala/akka/persistence/AtLeastOnceDeliveryCrashSpec.scala

```diff
@@ -75,6 +77,8 @@ class AtLeastOnceDeliveryCrashSpec
       testProbe.expectMsgType[CrashingActor.SendingMessage]
 
       superVisor ! CrashingActor.CrashMessage
+      testProbe.expectMsgType[CrashingActor.ConfirmCrashMessage.type]
```

Comment:
Ok, now that I better understand how this tests is working, I think I found the issue. 

In the logs, I found that `CrashMessage` was sent to dead-letter. That's because `superVisor` stopped before the message arriving and therefore on re-instantiation the actor will never crash and the `SendingMessage` will be redelivery which is not what this tests is expecting. 

Now we wait for a confirmation that the `CrashMessage` was delivered and persisted before we stop `superVisor`